utils/cronjobs_osgeo_lxd/cron_grass_current_stable_build_binaries.sh

```diff
@@ -8,36 +8,34 @@
 ###################################################################
 # how it works:
 # - it updates locally the GRASS source code from github server
-# - configures, compiles
+# - configures source code and then compiles it
 # - packages the binaries
 # - generated the install scripts
 # - generates the pyGRASS 8 HTML manual
 # - generates the user 8 HTML manuals
 # - injects DuckDuckGo search field
 
 # Preparations, on server (neteler@grasslxd:$):
-#  - Install PROJ incl Datum shift grids
-#  - Install GDAL
-#  - Install apt-get install texlive-latex-extra python3-sphinxcontrib.apidoc
-#  - Clone source from github:
-#    mkdir -p ~/src ; cd ~/src
-#    git clone https://github.com/OSGeo/grass.git releasebranch_8_4
-#    cd releasebranch_8_4
-#    git checkout releasebranch_8_4
-#  - Prepare target directories:
-#    cd /var/www/code_and_data/
-#    mkdir grass84
-#    cd /var/www/html/
-#    ln -s /var/www/code_and_data/grass84 .
+# - Install PROJ incl Datum shift grids
+# - Install GDAL
+# - Install apt-get install texlive-latex-extra python3-sphinxcontrib.apidoc
+# - install further dependencies:
+#     cd $HOME/src/releasebranch_8_4/ && git pull && sudo apt install $(cat .github/workflows/apt.txt)
+# - run this script
+# - one time only: cross-link code into web space on grasslxd server:
+#     cd /var/www/html/
+#     ln -s /var/www/code_and_data/grass84 .
 #
 #################################
 # variables for build environment (grass.osgeo.org specific)
-MAINDIR=/home/neteler
+MAINDIR=/home/$USER
```

Comment:
I tried to get this running in a Docker container, but $USER is not defined and /home is empty.

Reply:
Right, `$USER` [does not exist in docker](https://docs.docker.com/engine/containers/run/#environment-variables).

So I'll populate it with `id -u -n` (then the script work in and outside of docker) - done in a20fe94c0e6d50191f2cd2801807ae5c1ca8c12b